Sources/MapboxMaps/Annotations/AnnotationOrchestratorImpl.swift

```diff
@@ -53,14 +53,25 @@ internal final class AnnotationOrchestratorImpl: NSObject, AnnotationOrchestrato
         longPressGestureRecognizer.addTarget(self, action: #selector(handleDrag(_:)))
         longPressGestureRecognizer.delegate = self
         tapGestureRecognizer.delegate = self
+        longPressGestureRecognizer.isEnabled = false
+        tapGestureRecognizer.isEnabled = false
     }
 
     /// Dictionary of annotation managers keyed by their identifiers.
     internal var annotationManagersById: [String: AnnotationManager] {
         annotationManagersByIdInternal
     }
 
-    private var annotationManagersByIdInternal = [String: AnnotationManagerInternal]()
+    internal var annotationManagersByIdInternal = [String: AnnotationManagerInternal]() {
+        didSet {
+            if !annotationManagersById.isEmpty {
+                longPressGestureRecognizer.isEnabled = true
+                tapGestureRecognizer.isEnabled = true
+            }
+            longPressGestureRecognizer.isEnabled = !annotationManagersById.isEmpty
+            tapGestureRecognizer.isEnabled = !annotationManagersById.isEmpty
```

Comment:
I know there more tests coming for the annotation orchestrator implementation, but could you add tests for this newly added behavior of disabling gesture recognizers on init, and enabling/disabling them when managers get added/removed. Testing this shouldn't require major modifications(maybe `MapboxLongPressGestureRecognizer` would need to be mocked), there already are some tests in `AnnotationOrchestratorImplTests`.